### ISSUE TITLE
what the heck....

### ISSUE THREAD

**Issue by casigus** [TheGameCreators/GameGuruRepo#2382]
this happened.

![image](https://user-images.githubusercontent.com/68471279/156703405-ed0c4230-8d2a-4a49-8359-df38aeba4dc9.png)

then windows security deleted the ggmax exe.

**Comment by ArgentArts** [comment]
Dude, sorry, but do you know how to use GitHub? Titles here should not be things like, "what the heck...." and "this.", etc. GitHub is a place to report bugs and requests to help the developers of a software so they can ... well ... develop their software. "what the heck...." does not help them to do that. This is not a forum. 

This is something you should be looking up on Google, researching to see what it is and where you picked it up.

**Comment by casigus** [comment]
nit picking at a post.

this is a ggmax issue, as clearly shown in the screenshot.

GGMAX caused windows 10 defender to react, and the exe was removed.

if thats not a bug I dont know what is.


________________________________

**Comment by ArgentArts** [comment]
Yes, but none of the other Windows 10 users are reporting this as happening. So, why is that?
